packages/flutter_test/lib/src/binding.dart

```diff
@@ -948,32 +951,32 @@ class AutomatedTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
   bool get inTest => _currentFakeAsync != null;
 
   @override
-  int get microtaskCount => _currentFakeAsync.microtaskCount;
+  int get microtaskCount => _currentFakeAsync!.microtaskCount;
```

Comment:
So you can't get the current task count if not currently in a test, but do we want to throw?  Couldn't you just return zero if `!inTest`?

Reply:
I think that accessing this outside of a test is meaningless and almost certainly means you have some kind of a bug. Therefore, it should throw.

Reply:
OK, makes sense.